i18n/registration/tickets.i18n.js

```diff
@@ -0,0 +1,158 @@
+import { genI18nMessages } from '@/utils/i18n.utils'
+
+export default genI18nMessages({
+    'en-us': {
+        pageTitle: 'Online Ticket',
+        pageSummary:
+            'Welcome to participate in PyCon. You can purchase different ticket types, ' +
+            'and find the most suitable plan. If you need financial assistance, ' +
+            'you can go to the financial assistance application below. ',
+        individualTicket: {
+            title: 'Individual',
+            summary: {
+                text:
+                    'Tickets are limited. Be sure to get your ticket as soon as possible before they are sold out. ' +
+                    'We will send out electronic invoices by email after the conference. ' +
+                    '{highlight} All ticket sales end on September 4th. ',
+                pySafe:
+                    'The souvenirs of Pysafe include stickers, wine glasses, T-shirt and thankful card. ',
+            },
+            highlight: 'Individual tickets cannot use reimbursement. ',
+            type: [
+                {
+                    name: 'Regular',
+                    discount: 'Regular',
+                },
+                {
+                    name: 'Regular(with Pysafe)',
+                    discount: 'Regular',
+                },
+            ],
+        },
+        corporateTicket: {
+            title: 'Corporate',
+            summary: {
+                text:
+                    'Provide company name and Unified Business Number, ' +
+                    'or N/A if re­im­burse­ment is not required. ' +
+                    'We will send out electronic invoices by email after the conference. ' +
+                    '{groupDiscount}The sales of corporate tickets ended on September 4th. ',
+                pySafe:
+                    'The souvenirs of Pysafe include stickers, wine glasses, T-shirt and thankful card. ',
+            },
+            highlight: 'For group purchase, please contact us! ',
+            type: {
+                name: 'Corporate(with Pysafe)',
+                discount: 'Regular',
+            },
+            groupDiscount: {
+                text:
+                    'Group purchase discounts are provided for more than five people. ' +
+                    'For group purchase, please fill out the {formLink}! ',
+                formLink: 'Google form',
+            },
+        },
+        notices: {
+            title: 'Notices',
+            description: [
+                'If you need reimbursement, please purchase corporate ticket, ' +
+                    'other ticket cannot be provided. ',
+                'Tickets provide access to the main conference from 10 / 2 to 10 / 3. ' +
+                    'Please refer to their own pages for tutorial. ',
+                'Please provide valid means of contact (phone number and email) on purchase, ' +
+                    'so that we will be able to inform you of any important notices. ',
```

Comment:
```suggestion
                    "so that you won't be missed if we have important announcements. " +
                    'If you have any questions, please send us an e-mail to ' +
                    'organizers@pycon.tw, and we'll reply as soon as possible.'
```